[Package]: filelight

### Why is it worth to add this package?

The requested package is required for me in order to clean up my unwanted extra files from my termux workspace. This package provides me proper visualisation for my big files.

### Home page URL

https://apps.kde.org/filelight/

### Source code URL

https://github.com/KDE/filelight

### Packaging policy acknowledgement

- [X] The project is actively developed.
- [x] The project has [existing packages](https://repology.org/projects) and is "well known".
- [X] Licensed under an [open source license](https://spdx.org/licenses/).
- [x] Not available through a language package manager: pip, npm, cpan, cargo, etc.

- [X] Not taking up too much disk space (< 100MiB per architecture, exceptions can be made)
- [X] Not duplicating the functionality of existing packages.
- [X] Not serving hacking, malware, phishing, spamming, spying, ddos functionality.
- [X] I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied if it is found lacking.

### Additional information

I managed to build the package. Unfortunately I have encountered an error while building kirigami due to an ambigous error. When I ran it It told me it is unable to find the required qml modules provided by kconfig even though kf6-kconfig package is installed.

Error occured during builing kirigami
```
/data/data/com.termux/files/home/kirigami/src/kirigamiplugin.cpp:73:50: error: use of undeclared identifier 'baseUrl'
   73 |    Kirigami::Platform::StyleSelector::setBaseUrl(baseUrl());
      |                                                  ^
``` If this gets fixed I assume filelight will not give error and work properly

--- a/x11-packages/filelight/build.sh
+++ b/x11-packages/filelight/build.sh
@@ -1,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE="https://apps.kde.org/filelight/"
+TERMUX_PKG_DESCRIPTION="View disk usage information"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="24.08.2"
+TERMUX_PKG_SRCURL=https://download.kde.org/stable/release-service/${TERMUX_PKG_VERSION}/src/filelight-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=5e904cdf4573fa5b8578e4157da348ad46e7281992bfbd6af59c9430e16ba94e
+TERMUX_PKG_DEPENDS="kf6-kconfig, kf6-kcoreaddons, kf6-kcrash, kf6-ki18n, kf6-kio, kf6-kirigami, kf6-kirigami-addons, kf6-kwidgetsaddons, kf6-qqc2-desktop-style, libc++, qt6-qtbase, qt6-qtdeclarative"
+TERMUX_PKG_BUILD_DEPENDS="extra-cmake-modules"
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DBUILD_TESTING=OFF
+-DCMAKE_SYSTEM_NAME=Linux
+-DKDE_INSTALL_QMLDIR=lib/qt6/qml
+-DKDE_INSTALL_QTPLUGINDIR=lib/qt6/plugins
+"

--- a/x11-packages/kf6-kirigami-addons/build.sh
+++ b/x11-packages/kf6-kirigami-addons/build.sh
@@ -1,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE="https://invent.kde.org/libraries/kirigami-addons"
+TERMUX_PKG_DESCRIPTION="Add-ons for the Kirigami framework"
+TERMUX_PKG_LICENSE="LGPL-2.0, LGPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.5.0"
+TERMUX_PKG_SRCURL=https://download.kde.org/stable/kirigami-addons/kirigami-addons-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=f9dfd191022b633ea3348f84730d7158a159dfe49668dba0a9e9bd678ac44e54
+TERMUX_PKG_DEPENDS="kf6-kconfig, kf6-kcoreaddons, kf6-kglobalaccel, kf6-kguiaddons, kf6-ki18n, kf6-kirigami, libc++, qt6-qtbase, qt6-qtdeclarative"
+TERMUX_PKG_BUILD_DEPENDS="extra-cmake-modules"
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DBUILD_TESTING=OFF
+-DCMAKE_SYSTEM_NAME=Linux
+-DKDE_INSTALL_QMLDIR=lib/qt6/qml
+-DKDE_INSTALL_QTPLUGINDIR=lib/qt6/plugins
+"

